src/main/core.js

```diff
@@ -59,6 +51,7 @@ function attachComments(text, ast, opts) {
     comments.attach(astComments, ast, text, opts);
   }
   ast.tokens = [];
+  opts[Symbol.for("comments")] = astComments || [];
```

Comment:
Using `Symbol.for` instead of a common `Symbol("comments")`, because if we want use this in parsers, the parsers are bundled seperately, `Symbol("comments")` will be a different Symbol. Unless we add it to `pretter` object.